packages/shared/src/utils/route/route.ts

```diff
@@ -1,6 +1,9 @@
 // Checks if pathname matches route. (Works even with query string /?)
 // TODO: Add test cases for this
+import React from 'react';
+
 type TRoute = {
+    component?: typeof React.Component;
```

Comment:
```suggestion
    component?: React.ReactNode;
```

Reply:
Unfortunately, it doesn’t work. I had this error:
<img width="850" alt="Screenshot 2023-02-20 at 10 56 59" src="https://user-images.githubusercontent.com/103182683/220046030-a7bb4824-874c-4051-bd47-949c7b8c153f.png">

using 'typeof React.Component' I found on stackoverflow